### PR TITLE
[freeboxos] Fix thing status handling when rebooting the player

### LINKED ISSUE #17227: [freeboxos] Reboot player through action: thing status remains in DUTY_CYCLE
If I reboot the Revolution player through the OH action, the reboot of the player device is properly triggered but then the thing remains in status DUTY_CYCLE (System rebooting...) even after the reboot.
Restarting the binding or disabling/re-enabling the thing solves the issue.

### PR DESCRIPTION
**Pull request by lolodomo**
Fix #17227

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by clinique** [comment]
Is it so simple ?

**Review by clinique — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
> Is it so simple ?

The problem was that the Web socket does not detect and notify that the player was disconnected, probably because the cut of network is tooy short. Status was set to OFFLINE/DUTY_CYCLE when starting the reboot and due to the test that was based on thing status, the state was remaining OFFLINE for ever.

But I am going probably to restore the call to `internalPoll` when handling DUTY_CYCLE, it should be sufficient.

By the way, I am not totally satisfied with the thing status being changed by different conditions at twice places.

**Comment by clinique** [comment]
I think that when I implemented websockets I did not figure this kind of impact. Well discovered.

**Comment by lolodomo** [comment]
@clinique : ready for the review, I am now satisfied with the handling of thing status. And player reboot is working fine.

**Review by clinique — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java [hunk 1]
```diff
@@ -89,6 +101,9 @@ protected void internalPoll() throws FreeboxException {
                 }
             }
             updateChannelQuantity(SYS_INFO, UPTIME, uptime, Units.SECOND);
+        } else {
+            logger.debug("{}: poll with reachable={}", thing.getUID(), reachable);
+            updateStatus(ThingStatus.OFFLINE);
```
Comment (jlaur) [inline bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java:106]: Would it make sense to add a status description like "Player is not reachable" here?

**Review by jlaur — COMMENTED** [review]
Thanks! Just one question.

File: bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java]: Good idea, done.

File: bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java [hunk 2]
```diff
@@ -103,7 +108,7 @@ private void poll() throws FreeboxException {
             updateChannelQuantity(SYS_INFO, UPTIME, uptime, Units.SECOND);
         } else {
             logger.debug("{}: poll with reachable={}", thing.getUID(), reachable);
-            updateStatus(ThingStatus.OFFLINE);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, "Player is not reachable");
```
Comment (jlaur) [inline bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java:111]: I see there is already I18N support for status descriptions in the binding:
https://github.com/openhab/openhab-addons/blob/1f17e53bc2eb277b81c8da08e35aef58a928d3af/bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/FreeboxOsHandler.java#L74-L75

so maybe this one should support I18N as well?

Comment (lolodomo) [reply bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java]: Yes, done.

**Pull request merged**